workload_7_postprocess_fn/main.py

```diff
@@ -0,0 +1,186 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import functions_framework
+from google.cloud import monitoring_v3
+import statsmodels.stats.power as pwr
+import numpy as np
+import pandas as pd
+import time
+import os
+import scipy as sp
+
+
+# Subworkloads:
+# INSERT using InsertObject (Multipart Upload)
+# WRITE using WriteObject  (Resumable Upload)
+# READ method doesn't matter between XML / JSON
+WORKLOADS_LIST = [
+    "WRITE",
+    "INSERT",
+    "READ[0]",
+    "READ[1]",
+    "READ[2]",
+]
+HASH_TUPLE_LIST = [("1", "1"), ("0", "1"), ("1", "0"), ("0", "0")]
+
+
+def compare_mibs(df, a="Json", b="Xml", alpha=0.001):
```

Comment:
Do we have a doc mentioning what each of these tests do and stuff like, for example, why we chose 0.001 for alpha here? If not, can some of that be inlined here or added to the README?

Reply:
I had chatted with coryan on it; He selected it because it was more rigorous than 0.05 which is more commonly used. Added a comments to clarify the output as well

Reply:
Yes. The conventional alpha=0.05 (or 5%) is chosen by convention.  It makes more sense in fields where the experiments are expensive (e.g., medicine).  For us, where running a few more iterations is not that expensive, using a more stringent test is probably a good idea.